server/schedule/basic_cluster.go

```diff
@@ -200,14 +200,14 @@ func (bc *BasicCluster) PutRegion(region *core.RegionInfo) error {
 	return nil
 }
 
-// UpdateWriteStatus update the write status
-func (bc *BasicCluster) UpdateWriteStatus(region *core.RegionInfo) {
+// IsUpdateWriteStatus update the write status
+func (bc *BasicCluster) IsUpdateWriteStatus(region *core.RegionInfo) (bool, uint64, *core.RegionStat) {
```

Comment:
I suppose the second return value is always the region's id, we can load it from region info, so don't need to return it.